how to run ACCESS-OM3 and ACCESS-OM3 specification documentation (#893)
ACCESS-NRI/access-hive.org.au: docs/models/configurations/access-om.md

# ACCESS-OM

![ACCESS OM model](/assets/model-config-logos/configurations-without-titles/access-om.png){: class="img-contain white-background round-edges with-padding intro-img" loading="lazy"}

The ACCESS Ocean Model (ACCESS-OM) is a global coupled ocean model that includes [ocean](/models/model_components/ocean), [ocean biogeochemistry](/models/model_components/bgc_ocean), and [sea ice](/models/model_components/sea-ice) components, linked together by a [coupler](/models/model_components/coupler).<br>
The atmospheric fields that drive the model are provided by a data source, usually derived from reanalysis.

## ACCESS-OM3

ACCESS-OM3 is a suite of coupled ocean-sea ice models developed by ACCESS-NRI and the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)](http://cosima.org.au/).<br>

ACCESS-NRI will release the [ACCESS-OM3 configurations](https://github.com/ACCESS-NRI/access-om3-configs).

There are three planned global configurations for:

- 1° horizontal resolution, 50 vertical levels.
- 0.25° horizontal resolution, 50 vertical levels.

> - 0.12° horizontal resolution, 75 vertical levels.

Reviewer: 75 vertical levels for all

Author: Now fixed in [6ccbea4ab0e486a11e4391ea122d6205c4fca005](https://github.com/ACCESS-NRI/access-hive.org.au/pull/893/commits/6ccbea4ab0e486a11e4391ea122d6205c4fca005).

For each resolution there are **two planned options of atmospheric forcing**: 

- Repeat Year Forcing (RYF)

> - Interannual Forcing (IAF)

Reviewer: ```suggestion
- Inter-Annual Forcing (IAF)
```

Author: Now fixed in [6ccbea4ab0e486a11e4391ea122d6205c4fca005](https://github.com/ACCESS-NRI/access-hive.org.au/pull/893/commits/6ccbea4ab0e486a11e4391ea122d6205c4fca005).

> Each configuration also has a biogeochemical (BGC) configuration that uses the [Biogeochemistry Ocean component](/models/model_components/bgc_ocean), if this is required.

Reviewer: has -> can/will also have 

using "new" wombat - does new wombat (i.g. the generic tracers version) have a name ?

Author: @dougiesquire / @pearseb, please advise on this...

Reviewer 2: Can we just say "WOMBAT"? I don't think it's really relevant that it's the "new" WOMBAT and there may be multiple configs with different versions of WOMBAT: WOMBATlite, WOMBATmid etc.

If you do want to give a specific version, then WOMBATlite is the version that is currently being worked on.

Author: I agree, I think we keep it general.

Now rephrased in [6ccbea4ab0e486a11e4391ea122d6205c4fca005](https://github.com/ACCESS-NRI/access-hive.org.au/pull/893/commits/6ccbea4ab0e486a11e4391ea122d6205c4fca005).

!!! warning
    BGC experiments are slower and generally consume more resources (compute time and disk space).

### Model Components
<div class="tabLabels" label="ACCESS-OM3-versions">
    <button id="1deg">ACCESS-OM3</button>
    <button id="025deg">ACCESS-OM3-025</button>

>     <button id='01deg'>ACCESS-OM3-012</button>

Reviewer: I am not sure these names are useful.

I think it could be better to have "ACCESS-OM3" as the Model, and the resolutions/forcings as Configurations

Reviewer: If we want these names, then the first one should be ACCESS-OM3-1 and the last one ACCESS-OM3-010 or -008

Author: H'm, I think I agree it's a bit clunky. To be clear, I suggest looking at [the rendered version here](https://access-hive.org.au/pr-preview/pr-893/models/configurations/access-om/#access-om3).

How about changing `Model Components` to `Model Configurations`, I think we still need the names but they could relate to branch names? And then still list the model components and key features? The [om2 list](https://access-hive.org.au/pr-preview/pr-893/models/configurations/access-om/#access-om3) would need to be updated too.

Author: (I'm inclined to remove all references to the high-resolution config for the time being as that's some time away.)

Author: Now rephrased in [6ccbea4ab0e486a11e4391ea122d6205c4fca005](https://github.com/ACCESS-NRI/access-hive.org.au/pull/893/commits/6ccbea4ab0e486a11e4391ea122d6205c4fca005).

# ACCESS-OM

![ACCESS OM model](/assets/model-config-logos/configurations-without-titles/access-om.png){: class="img-contain white-background round-edges with-padding intro-img" loading="lazy"}

The ACCESS Ocean Model (ACCESS-OM) is a global coupled ocean model that includes [ocean](/models/model_components/ocean), [ocean biogeochemistry](/models/model_components/bgc_ocean), and [sea ice](/models/model_components/sea-ice) components, linked together by a [coupler](/models/model_components/coupler).<br>
The atmospheric fields that drive the model are provided by a data source, usually derived from reanalysis.

## ACCESS-OM3

ACCESS-OM3 is a suite of coupled ocean-sea ice models developed by ACCESS-NRI and the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)](http://cosima.org.au/).<br>

ACCESS-NRI will release the [ACCESS-OM3 configurations](https://github.com/ACCESS-NRI/access-om3-configs).

There are three planned global configurations for:

- 1° horizontal resolution, 75 vertical levels.

> - 0.25° horizontal resolution, 75 vertical levels.

Author: We've decided 0.25 (now called 25km) will be in the first release. So now there's only the one.

> - 0.25° horizontal resolution, 75 vertical levels.

Author: The OSIT team have decided to focus on the 0.25 / 25km configuration so won't be releasing the 1degree at the moment, the page has been updated to reflect this.

> - 0.25° horizontal resolution, 75 vertical levels.

Reviewer 3: I can still see the [OM3-1 configuration](https://access-hive.org.au/pr-preview/pr-893/models/configurations/access-om/#access-om3)

> - 0.25° horizontal resolution, 75 vertical levels.

Author: I got called away on a meeting! Should be here [35a1714](https://github.com/ACCESS-NRI/access-hive.org.au/pull/893/commits/35a17141ccc4a9bfc0d587bc7d23ed8bfea2f32e)

For each resolution there are **two options of JRA55-do atmospheric forcing**:

- Repeat Year Forcing (RYF): repeated 1 May 1990 - 30 April 1991 forcing
- Interannual Forcing (IAF): 1958-2018 forcing

Each configuration also has an optional biogeochemical (BGC) configuration that uses the [Biogeochemistry Ocean component](/models/model_components/bgc_ocean), if required.
!!! warning
    BGC experiments are slower and generally consume more resources (compute time and disk space).

> ### Model Configurations

Author: Best I can tell this is fixed in the latest commit. (Let me know if I missed it somehow.)

<div class="tabLabels" label="ACCESS-OM3-versions">
    <button id="1deg">ACCESS-OM3-1</button>

>     <button id="025deg">ACCESS-OM3-025</button>

Author: With the new branch naming scheme (25km), this issue has gone away

</div>

- **Ocean**: [MOM6](/models/model_components/ocean#mom6).<br>
  Tripolar grid, <span tabcontentfor="1deg">1</span><span tabcontentfor="025deg">0.25</span>° spatial resolution, 75 vertical levels.

- **Sea ice**: [CICE6](/models/model_components/sea-ice#cice6).<br>
    Same grid as ocean.

- **Ocean Biogeochemistry**: [WOMBAT](/models/model_components/bgc_ocean#wombat).

> 

Author: Done in the commit I'm about to push

> 

Reviewer 3: I unresolved the conversation because the commit has not been pushed yet.
Please resolve the conversation only when there is a pushed commit already (it can be useful also to add the commit link if possible) to avoid confusion.

> [cosima]: http://cosima.org

Reviewer 4: ```suggestion
[cosima]: https://cosima.org.au/
```

{% set model = "ACCESS-OM3" %}
## {{ model }}

{{ model }} is a suite of coupled ocean-sea ice models developed by ACCESS-NRI and the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)][cosima].

ACCESS-NRI has released [{{ model }} configurations](https://github.com/ACCESS-NRI/access-om3-configs).

There is a global configuration for **one spatial resolution**:

- 0.25° or 25km horizontal resolution, 75 vertical levels.

For the above resolution there is **one option of JRA55-do atmospheric forcing**:

- Repeat Year Forcing (RYF): repeated 1 May 1990 - 30 April 1991 forcing

Each configuration also has an optional biogeochemical (BGC) configuration that uses the [Biogeochemistry Ocean component](/models/model_components/bgc_ocean), if required.
!!! warning
    BGC experiments are slower and generally consume more resources (compute time and disk space).

### Model Components {: #model-components-{{model}} }

- **Ocean**: [MOM6](/models/model_components/ocean#mom6).<br>
  Tripolar grid.

- **Sea ice**: [CICE6](/models/model_components/sea-ice#cice6).<br>
    Same grid as ocean.

> - **Ocean Biogeochemistry**: [WOMBAT](/models/model_components/bgc_ocean#wombat).

Reviewer 2: ```suggestion
- **Ocean Biogeochemistry**: [WOMBAT](/models/model_components/bgc_ocean#wombat)  (optional).
```

The ACCESS Ocean Model (ACCESS-OM) is a global coupled ocean model that includes [ocean](/models/model_components/ocean), [ocean biogeochemistry](/models/model_components/bgc_ocean), and [sea ice](/models/model_components/sea-ice) components, linked together by a [coupler](/models/model_components/coupler).<br>
The atmospheric fields that drive the model are provided by a data source, usually derived from reanalysis.

{% set model = "ACCESS-OM3" %}
## {{ model }}

{{ model }} is a suite of coupled ocean-sea ice models developed by ACCESS-NRI and the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)][cosima].

ACCESS-NRI has released [{{ model }} configurations](https://github.com/ACCESS-NRI/access-om3-configs).

There is a global configuration for **one spatial resolution**:

- 0.25° or 25km horizontal resolution, 75 vertical levels.

For each resolution there are **two options of JRA55-do atmospheric forcing**:

- Repeat Year Forcing (RYF): repeated 1 May 1990 - 30 April 1991 forcing

> - Inter-Annual Forcing (IAF): 1958-2018 forcing

Reviewer: ```
- Inter-Annual Forcing (IAF): 1958-2018 forcing
```

This is not available at time of writing / release

Each configuration also has an optional biogeochemical (BGC) configuration that uses the [Biogeochemistry Ocean component](/models/model_components/bgc_ocean), if required.
!!! warning
    BGC experiments are slower and generally consume more resources (compute time and disk space).

### Model Components {: #model-components-{{model}} }

- **Ocean**: [MOM6](/models/model_components/ocean#mom6).<br>

>   Tripolar grid, 25 km spatial resolution, 75 vertical levels.

Author: Removed `25 km spatial resolution, 75 vertical levels.`

The ACCESS Ocean Model (ACCESS-OM) is a global coupled ocean model that includes [ocean](/models/model_components/ocean), [ocean biogeochemistry](/models/model_components/bgc_ocean), and [sea ice](/models/model_components/sea-ice) components, linked together by a [coupler](/models/model_components/coupler).<br>
The atmospheric fields that drive the model are provided by a data source, usually derived from reanalysis.

{% set model = "ACCESS-OM3" %}
## {{ model }}

{{ model }} is a suite of coupled ocean-sea ice models developed by ACCESS-NRI and the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)][cosima].

ACCESS-NRI has released [{{ model }} configurations](https://github.com/ACCESS-NRI/access-om3-configs).

> There is a global configurations for **one spatial resolutions**:

Reviewer 2: ```suggestion
There is a global configuration for **one spatial resolution**:
```

> - 0.25° or 25km horizontal resolution, 75 vertical levels.

Reviewer 2: Sorry, I know I commented on this last time, but here we say there's only 25km resolution, but in the "Run ACCESS-OM3" page we demonstrate running the 100km resolution. I think we need to be consistent between the two. The simplest fix is to also mention 100km resolution here:

```suggestion
- 1° or 100km horizontal nominal resolution, 75 vertical levels.
- 0.25° or 25km horizontal nominal resolution, 75 vertical levels.
```

Author: Fair point but thinking about this more, I don't think we're doing ourselves any future favours by doing it this way because the 100km will likely not be releasable by the time the 25km is. With that in mind, I've reverted the `docs/models/run-a-model/run-access-om3.md` page to using the 25km config'. See [9cc3eaa](https://github.com/ACCESS-NRI/access-hive.org.au/pull/893/commits/9cc3eaab0ba6b8f5c1f2219cff5c59bd7384ae2b)

For each resolution there are **two options of atmospheric forcing**: 

- Repeat Year Forcing (RYF)
- Inter-Annual Forcing (IAF)

Each configuration can also have a biogeochemical WOMBAT (BGC) addition (see [biogeochemistry ocean component](/models/model_components/bgc_ocean)), if this is required.
!!! warning
    BGC experiments are slower and generally consume more resources (compute time and disk space).

### Model Components
<div class="tabLabels" label="ACCESS-OM2-versions">

>     <button id="1deg">ACCESS-OM2-1</button>

Reviewer 2: There are issues with the spatial resolutions being rendered in this table. I also don't really understand what these tables add...

Author: Not sure what you mean by: `There are issues with the spatial resolutions being rendered in this table.`?

> I also don't really understand what these tables add...
I've had similar thoughts, I was looking to be consistent with the OM2 doc's.

Reviewer 2: > Not sure what you mean by: There are issues with the spatial resolutions being rendered in this table.?

I think @atteggiani has fixed this since I did my review

- **Coupler**: [OASIS3-MCT](/models/model_components/coupler#oasis3-mct).

> [Run ACCESS-OM](/models/run-a-model/run-access-om){: class="text-card"}

Reviewer 2: Should this be changed to `Run ACCESS-OM2`? And should a `Run ACCESS-OM3` button be added further up?

Author: I haven't edited those bits as @atteggiani is fixing that kind of problem (I think!?) with the new layout in the pipeline, see the new general layout preview [here](https://access-hive.org.au/pr-preview/pr-909/models/run-a-model/run-access-om/#ACCESS-OM3)
